tests/utilities/test_cli.py

```diff
@@ -673,13 +690,25 @@ def __init__(self, optim1: dict, optim2: dict, scheduler: dict):
             self.optim2 = instantiate_class(self.parameters(), optim2)
             self.scheduler = instantiate_class(self.optim1, scheduler)
 
-    cli_args = [
-        f"--trainer.default_root_dir={tmpdir}",
-        "--trainer.max_epochs=1",
-        "--optim2.class_path=torch.optim.SGD",
-        "--optim2.init_args.lr=0.01",
-        "--lr_scheduler.gamma=0.2",
-    ]
+    if use_registries:
+        cli_args = [
+            f"--trainer.default_root_dir={tmpdir}",
+            "--trainer.max_epochs=1",
+            "--optim1 Adam",
+            "--optim1.weight_decay 0.001",
```

Comment:
`sys.argv` splits on space, so these should be:
```suggestion
            "--optim1",
            "Adam",
            "--optim1.weight_decay",
            "0.001",
```

Reply:
Done !